solr/core/src/java/org/apache/solr/util/SolrCLI.java

```diff
@@ -1675,9 +1675,9 @@ protected void runImpl(CommandLine cli) throws Exception {
         }
 
         // convert raw JSON into user-friendly output
-        solrHome = (String)systemInfo.get("solr_home");
-        if (solrHome == null)
-          solrHome = configsetsDir.getParentFile().getAbsolutePath();
+        coreRootDirectory = (String)systemInfo.get("solr_core_root");
```

Comment:
Can you help my understand where exactly solr_core_root is populated?  I don't see it in SystemInfoHandler which I presume populates the response for this.

Reply:
It also had to be introduced, check SOLR-14878